packages/blockchain-link-types/src/messages.ts

```diff
@@ -130,7 +130,12 @@ export interface Unsubscribe {
 
 export interface PushTransaction {
     type: typeof MESSAGES.PUSH_TRANSACTION;
-    payload: string;
+    payload:
+        | string
```

Comment:
Why??? 😄 You have to add `getPushTransactionPayloadData` everywhere anyway, why not just change the type and avoid the extra complexity

Reply:
In Connect we do need to keep backwards compatibility in the API, but this is just a Blockchain link type, so there's nothing preventing us from changing it